backend_addon/{{ cookiecutter.__folder_name }}/README.md

```diff
@@ -19,6 +19,37 @@ And to create the Plone site:
 make create_site
 ```
 
+## Add features using `plonecli` or `bobtemplates.plone`
+
+This package provides markers as strings (`<!-- extra stuff goes here -->`) that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
+These markers act as hooks to add all kinds of subtemplates, including behaviors, control panels, upgrade steps, or other subtemplates from `plonecli`.
+
+To use them, you need to use `bobtemplates.plone` version X.X.X or later, and run the following command.
+
+```shell
+plonecli add -b .mrbob.ini content_type
+```
+
+The command passes the `.mrbob.ini` configuration file to `plonecli` to set some configuration variables which are needed to properly run the subtemplates.
+
+For example, you can add a behavior to your package with the following command.
+
+```shell
+plonecli add -b .mrbob.ini behavior
+```
+
+You can add a control panel with the following command.
+
+```shell
+plonecli add -b .mrbob.ini controlpanel
+```
+
+```{seealso}
+You can check the list of available subtemplates in the [`bobtemplates.plone` `README.md` file](https://github.com/plone/bobtemplates.plone/?tab=readme-ov-file#provided-subtemplates).
+See also the documentation of [Mockup and Patternslib](https://6.docs.plone.org/classic-ui/mockup.html) for how to build the UI toolkit for Classic UI.
```

Comment:
I opened a PR that needs to be reviewed and merged before this is finalized. See:

https://github.com/plone/documentation/pull/1734

Reply:
PR merged!